com.soulfiremc.soulfire.yml

```diff
@@ -0,0 +1,45 @@
+id: com.soulfiremc.soulfire
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: soulfire
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  # See https://github.com/tauri-apps/tauri/issues/9304
+  - --env=WEBKIT_DISABLE_DMABUF_RENDERER=1
+  # for Discord RPC mods
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
+  # Profile drag&drop
+  - --filesystem=xdg-download:ro
+  # Userspace visibility for manual hugepages configuration
+  # Required for -XX:+UseLargePages
+  - --filesystem=/sys/kernel/mm/hugepages:ro
+  # Userspace visibility for transparent hugepages configuration
+  # Required for -XX:+UseTransparentHugePages
+  - --filesystem=/sys/kernel/mm/transparent_hugepage:ro
+
+modules:
+  - shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json
+  - name: soulfire
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/AlexProgrammerDE/SoulFireClient/releases/download/1.7.1/SoulFire_1.7.1_amd64.deb
```

Comment:
so what happen when you build the aarch64 package? why not building from sources?

Reply:
Currently there's only amd64, I use the same approach as: https://github.com/flathub/com.kalbareader.Kalba/blob/ae04066a1fb9fdab02adc676af6e820b802b375a/com.kalbareader.Kalba.yml#L21

Reply:
you meant the same **incorrect** approach.

Tauri's doc is also incorrect. But it's not like they ever talked to us about it.

Reply:
saying you copied incorrectly doesn't solve the problem.

Reply:
I've added the only-arches `x86_64` to the flathub.json now just like kalba reader